src/components/delegate/delegate-section.tsx

```diff
@@ -0,0 +1,105 @@
+import { RVR_AUTHORIZER, RVR_TOKEN } from "@/constants/contracts"
+import { cn } from "@/lib/utils"
+import { useAccount, useReadContract, useReadContracts } from "wagmi"
+import { Typography } from "../ui/typography"
+import { WalletAddress } from "../wallet-address"
+import { formatUnits } from "viem"
+import { Skeleton } from "../ui/skeleton"
+import { DelegateForm } from "./delegate-form"
+import { AuthorizeClaimerForm } from "./authorize-claimer-form"
+
+export const DelegateSection = () => {
+  const { address } = useAccount()
+
+  const riverToken = useReadContracts({
+    allowFailure: false,
+    contracts: [
+      {
+        address: RVR_TOKEN.sepolia,
+        abi: RVR_TOKEN.abi,
+        functionName: 'balanceOf',
+        args: address ? [address] : undefined,
+      },
+      {
+        address: RVR_TOKEN.sepolia,
+        abi: RVR_TOKEN.abi,
+        functionName: 'decimals',
+      },
+    ],
+    query: {
+      enabled: !!address,
+    },
+  })
+
+  const delegatee = useReadContract({
+    address: RVR_TOKEN.sepolia,
+    abi: RVR_TOKEN.abi,
+    functionName: 'delegates',
+    args: address ? [address] : undefined,
+  })
+
+  const authorizedClaimer = useReadContract({
+    address: RVR_AUTHORIZER.sepolia,
+    abi: RVR_AUTHORIZER.abi,
+    functionName: 'getAuthorizedClaimer',
+    args: address ? [address] : undefined,
+  })
+
+  return (
+    <section
+      className={cn(
+        'hero-glow relative w-full overflow-x-clip bg-gray-90 md:min-h-screen',
+        'flex w-full flex-col items-center',
+        'pb-24 pt-[88px]',
+      )}
+    >
+      <section className={cn('flex w-full flex-col gap-6 px-4', 'sm:max-w-lg')}>
+        <Typography
+          as="h1"
+          size="3xl"
+          className="text-2xl font-semibold leading-[44px] text-gray-10"
+        >
+          Delegate
```

Comment:
Should we add some copy below this to describe what this form achieves for the user ? It shouldn't be assumed that user's know what "Delegate" does without a plain language description.